src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Razor/ComponentAccessibilityCodeActionProvider.cs

```diff
@@ -137,7 +137,8 @@ private void AddComponentAccessFromTag(RazorCodeActionContext context, MarkupSta
                 continue;
             }
 
-            var fullyQualifiedName = tagHelperPair._short.Name;
+            // if fqn contains a generic typeparam, we should strip it out
```

Comment:
Nit: Could probably expand this comment to include the "why", or perhaps just an example of what would happen if we don't, just to aid future explorers.